pose_format/pose.py

```diff
@@ -112,8 +112,13 @@ def get_components(self, components: List[str], points: Dict[str, List[str]] = N
                 new_component = PoseHeaderComponent(component.name, component.points,
                                                     component.limbs, component.colors, component.format)
                 if points is not None and component.name in points:  # copy and permute points
+                    # print(component.points)
+                    # print(component.limbs)
                     new_component.points = points[component.name]
                     indexes[component.name] = [component.points.index(p) for p in new_component.points]
+                    new_component.limbs = [(a, b) for a, b in new_component.limbs if str(a) in new_component.points and str(b) in new_component.points]
```

Comment:
> And only keep limbs that have points left.

@AmitMY this is what I intend to do the job, do you think it's the correct way? Anyway it leads the error.

Reply:
But I am not sure the error is due to the limbs, don't quite understand the logic there.